src/main/webapp/js/diagramly/EditorUi.js

```diff
@@ -12857,6 +12857,14 @@
 							data = data.xml;
 						}						
 					}
+					else if (data.action == 'Save'){
+						this.actions.get('save').funct();
+						return;
+					}
+					else if (data.action == 'exit'){
+						this.actions.get('exit').funct();
+						return;
+					}
```

Comment:
Why the Uppercase on "Save" and not "exit" ?
it must be the same, either capital or not

Reply:
i don't know why but drawio doesn't detecte 'save', or it get called in another place
so i meke it Uppercase
ok, i will change Exit

Reply:
If this is a drawio constraint, it may not be necessary to change the whole code to move away from the original. Leave it like that I think.

Reply:
i change Exit to Uppercase